app/services/channels/approve_channel_transfer.rb

```diff
@@ -1,35 +1,28 @@
 module Channels
   class ApproveChannelTransfer < BaseService
 
-    def initialize(channel:, should_delete: true)
+    def initialize(channel:)
       @channel = channel
-      @should_delete = should_delete
     end
 
     def perform
       ActiveRecord::Base.transaction do
         contested_by = @channel.contested_by_channel
-        contested_by.verified = true
-        contested_by.verification_pending = false
-        contested_by.save!
-        
-        # Verify the contested_by channel
-        original_owner_email = @channel.publisher.email
-        channel_name = @channel.publication_title
-
-        # Unverify and destroy original channel
-        @channel.verified = false
         @channel.contested_by_channel_id = nil
         @channel.contest_token = nil
         @channel.contest_timesout_at = nil
         @channel.save!
-        
+
         original_owner_email = @channel.publisher.email
         original_owner_name = @channel.publisher.name
         channel_name = @channel.publication_title
 
         # Delete the channel from eyeshade and clean up the promo registration
-        DeletePublisherChannelJob.perform_now(channel_id: @channel.id) if @should_delete
+        DeletePublisherChannelJob.perform_now(channel_id: @channel.id)
```

Comment:
My gut says we should move this line above the ActiveRecord::Transaction. Thoughts?